### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by Daniela-git**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
-    "outDir": "dist",
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "noUnusedParameters": true,
-    "noUnusedLocals": true
+      "outDir": "dist",
+      "noUnusedLocals": true
   }
-}
+}
```
Comment (germandavid85) [inline tsconfig.json]: let's add an additional line here

File: package.json [hunk 2]
```diff
@@ -9,7 +9,9 @@
     "test:local": "npm run build && protractor dist/protractor/local.config.js",
     "test:headless": "npm run build && protractor dist/protractor/headless.config.js",
     "test": "npm run build && protractor dist/protractor/saucelabs.config.js",
+    "webdriver:update": "webdriver-manager update --gecko false",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+
```
Comment (germandavid85) [inline package.json]: extra blank line missing

File: protractor/local.config.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { reporter } from "./helpers/reporter";
 
 export const config: Config = {
   framework: "jasmine",
-  specs: ["../test/**/*.spec.js"],
+  specs: ["../test/**/**.spec.js"],
```
Comment (germandavid85) [inline protractor/local.config.ts]: is this extra `*` needed?

**Review by germandavid85 — CHANGES_REQUESTED** [review]
this point of the workshop is about configuring travis but I don't see either the travis file or the GitHub check running in this pull request

**Comment by Daniela-git** [comment]
> this point of the workshop is about configuring travis but I don't see either the travis file or the GitHub check running in this pull request

My bad, I don't know what I did. Now the Travis' CI is activated every time

File: .nvmrc [hunk 4]
```diff
@@ -0,0 +1 @@
+v15.3.0
```
Comment (holgiosalos) [inline .nvmrc]: EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon: 
image

File: tsconfig.json [hunk 5]
```diff
@@ -1,11 +1,7 @@
...
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "noUnusedParameters": true,
-    "noUnusedLocals": true
+      "outDir": "dist",
+      "noUnusedLocals": true
   }
-}
+  
+}
```
Comment (holgiosalos) [inline tsconfig.json]: EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon: 
image

File: .nvmrc (hunk 4 above)
Comment (Daniela-git) [reply .nvmrc]: in which case is this necessary? or always?

**Review by holgiosalos — APPROVED** [review]

File: .nvmrc (hunk 4 above)
Comment (holgiosalos) [reply .nvmrc]: all files should have blank line as the end of file

**Review by holgiosalos — APPROVED** [review]

**Review by Daniela-git — COMMENTED** [review]
done

**Pull request merged**